docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -831,6 +867,9 @@ Default stratifications to all observers should include scenario and input draw.
       * Misoprostol coverage
       * Hemoglobin screening coverage
       * Ferritin screening coverage
+      * IFA coverage
+      * MMS coverage
```

Comment:
If I'm right that IFA and MMS are mutually exclusive, this should be a single stratification

Reply:
I like 2 columns